README.md

```diff
@@ -804,6 +807,8 @@ Function Name | Prop       | CSS Property    | Theme Field  | Responsive
 `justifySelf` | `justifySelf` | `justify-self` | none | yes
 `alignSelf` | `alignSelf` | `align-self` | none | yes
 `order` | `order` | `order` | none | yes
+`flexWrap` | `flexWrap` | `flex-wrap` | none | yes
+`flexBasis` | `flexBasis` | `flex-basis` | none | yes
```

Comment:
We should s/none/width/ since we're using the `getWidth` getter.

Reply:
👍